Documentation/performance.tex

```diff
@@ -0,0 +1,79 @@
+
+\begin{table}[!ht]
+    \centering
+    \caption{Properties used in the performance assessment of ESPO-R5v1.0}
+    \begin{tabular}{l|c|c|c|l}
+    \hline
+    Property                               & Short name & Variables          & Aspect   & Source \\ \hline
+    Mean                                   & mean       & tasmin, tasmax, pr & marginal & VALUE \\ \hline
+    First percentile                       & q01        & tasmin, tasmax     & marginal & VALUE \\ \hline
+    95th percentile                        & q95        & pr                 & marginal & VALUE \\ \hline
+    99th percentile                        & q99        & tasmin, tasmax, pr & marginal & VALUE \\ \hline
+    Dry spell frequency                    & dry\_spell\_freq & pr           & marginal & VALUE \\ \hline
+    Amplitude of the annual cycle          & aca        & tasmin, tasmax     & temporal & VALUE \\ \hline
+    Relative amplitude of the annual cycle & aca        & pr                 & temporal & VALUE \\ \hline
+    Lag-1 autocorrelation                  & lag1       & tasmax             & temporal & VALUE \\ \hline
+    Intervariable correlation (spearman)   & corr       & tasmin-tasmax, pr-tasmax & multivariate & VALUE \\ \hline
+    Spatial correlograms                   & correlogram & tasmin, tasmax, pr & spatial & \cite{Francois2020} \\\hline
+    First EOF                              & first\_eof & tasmin, tasmax, pr & spatial & \cite{Vrac2018} \\ \hline
+    \end{tabular}
+    \label{tab:props}
+
+\end{table}
+
+The \emph{properties} were computed on the calibration period (1981-2010) on the raw simulations, on the bias-adjusted scenarios and on the reference. The results for the raw simulations were compared to those for the reference using a \emph{measure} appropriate for that each property.\footnote{All properties in \ref{tab:props} are compared using a simple bias measure ($P_{sim} - P_{ref}$) except for the \emph{relative amplitude of the annual cycle}, which uses the ratio measure ($P_{sim} / P_{ref}$)}. This way, for each member of the ensemble, one can visualize a given property and compare the measures to assess the improvement brought by the bias adjustment. Because of performance issues, the properties and measures were not computed on the full domain of ESPO-R, but rather on three smaller regions chosen to cover different climate types.
```

Comment:
```suggestion
The \emph{properties} were computed on the calibration period (1981-2010) on the raw simulations, on the bias-adjusted scenarios and on the reference (ERA5-Land). The results for the raw simulations were compared to those for the reference using a \emph{measure} appropriate for that each property.\footnote{All properties in \ref{tab:props} are compared using a simple bias measure ($P_{sim} - P_{ref}$) except for the \emph{relative amplitude of the annual cycle}, which uses the ratio measure ($P_{sim} / P_{ref}$)}. This way, for each member of the ensemble, one can visualize a given property and compare the measures to assess the improvement brought by the bias adjustment. Because of performance issues, the properties and measures were not computed on the full domain of ESPO-R, but rather on three smaller regions chosen to cover different climate types.
```